plugins/formattingFilters/dateAndTime/DateTimeFilters.ts

```diff
@@ -0,0 +1,40 @@
+import Vue from 'vue'
+import VueMoment from 'vue-moment'
+import moment from 'moment'
+
+Vue.use(VueMoment, { moment })
+
+export class DateTimeFilters {
+  formatMilliToDateTime(value) {
+    if (value === null) {
+      // since null == undefined
```

Comment:
What I meant was use `value === null || value === undefined` instead, and remove the comment

Reply:
Missed out on the comment. Fixed.